### ISSUE TITLE
Any $ operators signal syntax error

### ISSUE THREAD

**Issue by 0zkr1** [Microsoft/vscode-cosmosdb#703]
Repro steps:

In VS Code using Azure Cosmo DB extension attached to a loco instance of Mongo, create a Mongo Scrapbook, and submit any command using $ operator. For example:

`
db.Users.find({
   "user": {
      $in: [ "A80", "HPA" ]
   }
},{
   "_id": false
});
`

Action: cosmosDB.executeMongoCommand
Error type: Error
Error Message: Error near line 0, column 30, text **'$in'**. Please check syntax.

At the Mongo Shell that command execute with no errors and with the correct results. I'm using an local Mongo instance installed in the same development machine at default port. I have configured the shell path:

`"mongo.shell.path": "C:/Program Files/MongoDB/Server/3.6/bin/mongo.exe"`

**What I'm missing for this kind of operators work?**

Version: 0.7.2
OS: win32
Mongo: 3.6
VSCode: 1.24.1

**Comment by PrashanthCorp** [comment]
Thanks for the feedback! 
You aren't missing anything regarding the operators. Currently, we only support double-quoted strings. 
We're working on making strings work as is. 
See #466 and #467. 

Please try placing double quotes around the strings and let us know if it works. :-)

**Comment by 0zkr1** [comment]
Thank you so much PrashanthCorp: I read it yesterday, but I can't answered it.

And yes, I tried all the combinations of quotes before registering the ticket, including around the $ operator (like you pointed at #688  or #691): same syntax error message or `<missing '('>`.

**Comment by StephenWeatherford** [comment]
@0zkr1 Adding double quotes around the "$" works for me in 0.7.2:

```
db.Users.find({ "user": { "$in": [ "A80", "HPA" ] } },{ "_id": false });
```

**Comment by PrashanthCorp** [comment]
Looks fixed by the PR #729

**Comment by PrashanthCorp** [comment]
Hi @0zkr1,  the merged fix was published today (version 0.8.0). Hope this fixes your issue. :-)
Let us know how the fix holds up with your use-case.
